internal/metamorph/processor.go

```diff
@@ -36,6 +36,9 @@ const (
 
 	processStatusUpdatesIntervalDefault  = 500 * time.Millisecond
 	processStatusUpdatesBatchSizeDefault = 1000
+
+	processTransactionsBatchSizeDefault = 200
+	processTransactionsIntervalDefault  = 2 * time.Second
```

Comment:
Are you sure it is not to long interval?

Reply:
I changed it to 1s for now. We can later make a setting of it and tweak the value